lib/counters/inMemory.js

```diff
@@ -1,15 +1,17 @@
 var _ = require('lodash');
-var lruCache = require('lru-cache');
+var LRUCache = require('lru-cache');
 
 module.exports = function init(options) {
-  var size = _.get(options, 'size') || 1000;
-  var cache = lruCache({ max: size });
+  const size = _.get(options, 'size') || 1000;
+  const cache = new LRUCache({
+    max: size,
+  });
 
   return {
-    incrementAndGet: function(key, next) {
+    incrementAndGet: function (key, next) {
```

Comment:
The in-house style is no space between in `function (`. I'll update the linter to fix this automatically from now on.